### DISCUSSION TITLE
想知道收费标准是什么？试用了一下余额是负数，是还可以欠吗？

### DISCUSSION THREAD

**Discussion by inzer99011** [ZuodaoTech/everyone-can-use-english#401] [General]
想知道收费标准是什么？试用了一下余额是负数，是还可以欠吗？

**Comment by an-lee** [comment]
负数就不能继续用了。按量收费。

**Reply by chen-zhaojin** [reply]
我的也是因为负数不能用了，请问老师在哪里可以充值，Enjoy上的充值没开放

**Reply by an-lee** [reply]
> 我的也是因为负数不能用了，请问老师在哪里可以充值，Enjoy上的充值没开放

充值开放了呀，你还没升级到最新版本吧。

**Comment by lishan256** [comment]
我也想知道，收费标准是什么。不然做什么要花多少银子，心里没底...

**Comment by mosaiclee** [comment]
怎么加载本地模型？我下载后好像一直加载不成功
